src/components/CowPen/CowPen.sass

```diff
@@ -29,6 +25,9 @@
       z-index: -20
 
     &.is-selected
+      // Render the selected cow on top of all others
+      z-index: 40
```

Comment:
z-indexes are my mortal enemy haha. i strongly believe 0, 1, and -1 are the only acceptable values, but it can take some reworking to get to this point. not sure if it's something you care about, but if it is we can work on taming these sometime

Reply:
I forgot where I learned to use factors of 20 for z-index, but it's served me reasonably well over the years. I think your CSS skills are quite a bit stronger than mine, so I'm happy to defer to your judgment for these sorts of things. Feel free to wrangle these as you see fit! 🙂